lib/authc/OAuthBasicExchangeAuthenticator.js

```diff
@@ -42,10 +44,18 @@ OAuthBasicExchangeAuthenticator.prototype.authenticate = function authenticate(c
         (apiKey.account.status==='ENABLED')
       ){
         var result = new AuthenticationResult(apiKey,self.application.dataStore);
+
         result.forApiKey = apiKey;
         result.application = self.application;
-        result.tokenResponse = self.buildTokenResponse(apiKey);
-        callback(null,result);
+
+        self.buildTokenResponse(apiKey, function (err, tokenResponse) {
```

Comment:
Depends on what style that you are referring to as our current:) The whole style being used in the project is very inconsistent. So for that reason I'm not adhering to it until we have it speced in eslint. And the formatting I currently follow is the one that we're going to be using.